### DISCUSSION TITLE
How can I help?

### DISCUSSION THREAD

**Discussion by felipeschneider88** [dataplat/DataSaturdays#117] [General]
Hello, my name is Felipe Schneider. I'm a SQL Server DBA but also got plenty of experience with .NET, python and JAVA. How can I help? There is some TODO list or backlog?

**Comment by way0utwest** [comment]
As of now, we have some issues, many of which likely need to be discussions and not code issues. I don't think we have a good architecture or direction as of yet. Perhaps you want to open a discussion and lead some debate on how to structure a toolset?
